plugins/modules/win_dns_record.py

```diff
@@ -77,12 +77,13 @@
     - The zone must already exist.
     required: yes
     type: str
-  computer_name:
+  server:
     description:
       - Specifies a DNS server.
       - You can specify an IP address or any value that resolves to an IP
         address, such as a fully qualified domain name (FQDN), host name, or
         NETBIOS name.
+    version_added: 1.2.0
```

Comment:
Sorry I was not aware that `computer_name` was already in use here, we still need to accept as an alias that for backwards compatibility. We can skip the `version_added` as technically the option isn't new, just make sure the description mentions it was renamed in the 1.2.0 release like so

```suggestion
  server:
    description:
      - Specifies a DNS server.
      - You can specify an IP address or any value that resolves to an IP
        address, such as a fully qualified domain name (FQDN), host name, or
        NETBIOS name.
      - Was renamed from I(computer_name) to I(server) in 1.2.0.
    aliases:
    - computer_name
```